content/en/news/releases/1.14.x/announcing-1.14.5/index.md

```diff
@@ -7,10 +7,14 @@ publishdate: 2022-10-11
 release: 1.14.5
 ---
 
-This release includes security fixes in Go 1.18.7 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
-This release also includes fixes to improve robustness.
+This release contains a fix for [CVE-2022-39278](/news/security/istio-security-2022-007/#cve-2022-39278) and
+bug fixes to improve robustness. We recommend users install this release instead of Istio 1.14.4,
+which does not contain the above CVE fix.
 This release note describes what is different between Istio 1.14.4 and Istio 1.14.5.
 
+FYI, This release includes security fixes in Go 1.18.7 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
+This release also includes fixes to improve robustness.
```

Comment:
"This release also includes fixes to improve robustness." can be removed because it is mentioned in the line above.